Date: Thu, 27 Jul 2023 18:49:29 +0000
Subject: ip: Bump default ttl to 127.

In 4.14 and 5.4, the default TTL was bumped up to 255, but we moved
the change to sysctl-defaults package.  However, sysctl config is not
applied to netns.

Let's bump it again but to 127 as some nodes could block packets with
TTL 255.

Signed-off-by: Kuniyuki Iwashima <kuniyu@amazon.com>

diff --git a/include/uapi/linux/ip.h b/include/uapi/linux/ip.h
--- a/include/uapi/linux/ip.h
+++ b/include/uapi/linux/ip.h
@@ -67,7 +67,7 @@
 
 #define IPVERSION	4
 #define MAXTTL		255
-#define IPDEFTTL	64
+#define IPDEFTTL	127
 
 #define IPOPT_OPTVAL 0
 #define IPOPT_OLEN   1
